docroot/profiles/favrskov/themes/favrskovtheme/template/Views/job/views-view-table--job--job-listing-by-table.tpl.php

```diff
@@ -19,17 +19,20 @@
  * @ingroup views_templates
  */
 ?>
+<?php $categories = array(); ?>
+<?php foreach ($rows as $row_count => $row): ?>
+  <?php
+    $categories[] = $row['field_job_category'];
+  ?>
+<?php endforeach; ?>
 <table <?php if ($classes) { print 'class="'. $classes . '" '; } ?><?php print $attributes; ?>>
-  <?php if (!empty($title) || !empty($caption)) : ?>
-    <caption><?php print $caption . $title; ?></caption>
-  <?php endif; ?>
   <?php if (!empty($header)) : ?>
     <thead>
     <tr>
       <?php unset($header['field_job_destination_link']); ?>
       <?php foreach ($header as $field => $label): ?>
         <th <?php if ($header_classes[$field]) { print 'class="'. $header_classes[$field] . '" '; } ?> scope="col">
-          <?php print $label; ?>
+          <span style="color: #fff;"><?php if($field == 'title') {print $title . " " . "(" . count($categories) . ")";} else {print $label;} ?></span>
```

Comment:
Please avoid such practice to insert style tags directly unless it is 100% required. Add an id or class. 
Another good practice is to generate end strings in some preprocess function: $title . " " . "(" . count($categories) . ")" - this can be easily moved out in template.php or somewhere else and here to print only the variable.

Reply:
I've fixed this. Moved style to scss file and moved the manipulation of the output in template.php.